Library/Formula/pandoc.rb

```diff
@@ -24,7 +24,12 @@ class Pandoc < Formula
 
   def install
     cabal_sandbox do
-      cabal_install "--only-dependencies"
+      args = []
+      if MacOS.version <= :lion
+        # disable AESNI for OSX <= 10.7
+        args << "--constraint=cryptonite -support_aesni"
```

Comment:
Can do this in one line: `args << "--constraint=cryptonite" << "-support_aesni" if MacOS.version <= :lion` and skip the comment as it's self-explanatory from the code.